obspy/io/gse2/src/GSE_UTI/gse_functions.c

```diff
@@ -231,7 +231,7 @@ int decomp_6b_buffer (int n_of_samples, int32_t *dta, char * (* reader)(char *,
   	if (ibuf > 79 || isspace(cbuf[ibuf])) { 
   	if ((*reader)(cbuf, vptr) == NULL) /* get next line */
   		{fprintf (stderr, "decomp_6b: missing input line?\n"); return -1; }
-      /* We need a space to be sure that CHK2 is not occuring in the middle
+      /* We need a space to be sure that CHK2 is not occurring in the middle
        * of the encoded string/buffer */
```

Comment:
This is mostly vendored but we made some changes, and I guess the changes can stay in